api/src/product/product.controller.ts

```diff
@@ -1,4 +1,7 @@
+<<<<<<< HEAD
 import { JwtGuard } from './../auth/guards/jwt.guard';
+=======
+>>>>>>> 98f725c (NestJS and Mongo CRUD operations)
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Unresolved merge conflict in controller</b></div>
<div id="fix">

There appears to be an unresolved merge conflict in the code. Consider resolving this conflict before proceeding with the PR.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -14,4 +14,1 @@
 -<<<<<<< HEAD
 -  UseGuards,
 -=======
 ->>>>>>> 98f725c (NestJS and Mongo CRUD operations)
 +  UseGuards,
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/omkardongre/amazon-clone/pull/3#issuecomment-2466190444>#8956f5</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged